pkg/volume/util/util.go

```diff
@@ -96,29 +97,34 @@ func UnmountPath(mountPath string, mounter mount.Interface) error {
 // IsNotMountPoint will be called instead of IsLikelyNotMountPoint.
 // IsNotMountPoint is more expensive but properly handles bind mounts.
 func UnmountMountPoint(mountPath string, mounter mount.Interface, extensiveMountPointCheck bool) error {
+	isStaleConnection := false
 	if pathExists, pathErr := PathExists(mountPath); pathErr != nil {
-		return fmt.Errorf("Error checking if path exists: %v", pathErr)
+		isStaleConnection = IsStaleConnection(pathErr)
+		if !isStaleConnection {
+			return fmt.Errorf("Error checking if path exists: %v", pathErr)
+		}
 	} else if !pathExists {
 		glog.Warningf("Warning: Unmount skipped because path does not exist: %v", mountPath)
 		return nil
 	}
 
-	var notMnt bool
-	var err error
-
-	if extensiveMountPointCheck {
-		notMnt, err = mount.IsNotMountPoint(mounter, mountPath)
-	} else {
-		notMnt, err = mounter.IsLikelyNotMountPoint(mountPath)
-	}
+	if !isStaleConnection {
+		var notMnt bool
```

Comment:
Wrong format here?

Tab?

Reply:
Yes ,it is a Tab. Tab is not allowed? 
I have used go fmt to format it